File: Java/Tests/jdi-uitest-wintests/src/main/java/com/epam/jdi/uitests/testing/unittests/pageobjects/w3cSite/W3cSite.java
package com.epam.jdi.uitests.testing.unittests.pageobjects.w3cSite;

import com.epam.jdi.uitests.win.winium.elements.composite.WebSite;
import com.epam.jdi.uitests.win.winium.elements.pageobjects.annotations.JPage;
import com.epam.jdi.uitests.win.winium.elements.pageobjects.annotations.JSite;

/**
 * Created by Maksim_Palchevskii on 9/10/2015.
 */

@JSite(domain = "http://www.w3schools.com")
public class W3cSite extends WebSite {
    @JPage(url = "/tags/tag_button.asp")
    public static FramePage framePage;

}

File: Python/Test/jdi_uitests_webtests/main/page_objects/w3c_site/w3c_site.py
from JDI.web.selenium.elements.composite.web_site import WebSite
from Test.jdi_uitests_webtests.main.page_objects.w3c_site.frame_page import FramePage


class W3cSite(WebSite):
    domain = "http://www.w3schools.com"

    frame_page = FramePage(url="/tags/tag_button.asp", domain=domain)